### ISSUE TITLE
Pagination support

### ISSUE THREAD

**Issue by peroumal1** [andrewalker/p5-webservice-digitalocean#7]
Hello, 

I went through the docs on MetaCPAN, and it looks like pagination is not supported by the module. Can you confirm ? 

Regards

**Comment by andrewalker** [comment]
Yes, I can confirm. I haven't needed pagination in the module, so I didn't implement it (yet). Patches welcome :)
